taichi/runtime/llvm/llvm_context.h

```diff
@@ -43,6 +42,8 @@ class TaichiLLVMContext {
   // main_thread is defined to be the thread that runs the initializer
   JITModule *runtime_jit_module{nullptr};
 
+  std::unique_ptr<ThreadLocalData> link_context_data{nullptr};
```

Comment:
```suggestion
  std::unique_ptr<ThreadLocalData> linked_context_data{nullptr};
```

Reply:
It's a ThreadLocalData containing the context for linking so maybe `linking_context_data` sounds better?

Reply:
or simpler `linking_context` :P yea I'm open to either this or `linking_context_data` if you prefer ;)